trymito.io/excel-to-python-page-contents/types.tsx

```diff
@@ -0,0 +1,41 @@
+export type PageContent = {
```

Comment:
Could leave a comment that says `see example in folder`, but I think this is fine